First go at frugal-basicwardrobe

jinpark opened the pull request:
Check it out and tell me if you think you want anything changed. Also when you upload it, let me know so I can post it to r/frugalmalefashion :)

-Jin

jinpark committed:
First go at frugal-basicwardrobe

taylorlapeyre commented:
It's a good start, but it needs some improvements.
1. Images should all be the same size and proportions.
2. If the images are going to change, then they will need to reflect the difference between the two sites (i.e. don't put any expensive clothes as the images)
3. Same thing with the text - it should discuss the clothes as it relates to the frugal side of things instead of rephrasing the original information.
4. The three paragraphs are a little much. Most of the appeal of the original site was that it kept the information very clear and concise. Try to make it one paragraph or restructure the information hierarchy to make the intro text and explanation separate.

jinpark committed:
Updated image sizing, redid descriptions, changed a few items

jinpark commented:
Hi,

I updated the image sizing and I'm not sure what you're talking about with the expensive clothes in the images. All the images are of things I listed in the frugal guide. Changed the text but theres only so much I can do with talking about frugality, it all ends up being the same things. Reduced the paragraphs in the beginning.

taylorlapeyre commented:
Just published at http://basicwardrobe.info/frugal/
